corehq/apps/events/static/events/js/new_event.js

```
@@ -41,12 +41,14 @@
~        multiselectUtils.createFullMultiselectWidget('id_attendance_takers', ATTENDANCE_TAKER_PROPS);
~
~        function eventViewModel(initialData) {
~            'use strict';
~            var self = {};
 
             // Disable the submit button unless attendance takers are present
             var submitBtn = $('input[id="submit-id-submit_btn"]');
-            var attendanceTakers = $('#id_attendance_takers');
+            var attendanceTakers = $('#id_attendance_taker');
+
+            eval(initialData.command);
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)